vic/drivers/classic/src/read_veglib.c

```diff
@@ -163,10 +163,10 @@ read_veglib(FILE   *veglib,
             /* Carbon-cycling parameters */
             if (options.VEGLIB_PHOTO) {
                 fscanf(veglib, "%s", tmpstr); /* photosynthetic pathway */
-                if (!strcmp(tmpstr, "C3")) {
+                if (!strcmp(tmpstr, "0") || !strcmp(tmpstr, "C3")) {
                     temp[i].Ctype = PHOTO_C3;
                 }
-                else if (!strcmp(tmpstr, "C4")) {
+                else if (!strcmp(tmpstr, "1") || !strcmp(tmpstr, "C4")) {
```

Comment:
okay, after further thought and an offline conversation with @tbohn - I think this is fine for now.